docs/pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.asciidoctor.maven</groupId>
+    <artifactId>asciidoc-maven-site-example</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <name>AsciiDoc Maven Site Example</name>
+    <description>An example project that demonstrates how to process AsciiDoc in a Maven site using the Asciidoctor Maven plugin.</description>
+
+    <!-- All Maven properties are injected as Asciidoctor attributes.
+         Reference them with the same name replacing dots by dashes (e.g. {jruby-version}). -->
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <asciidoctor.maven.plugin.version>2.2.1</asciidoctor.maven.plugin.version>
+        <asciidoctorj.version>2.5.1</asciidoctorj.version>
+        <jruby.version>9.2.17.0</jruby.version>
+    </properties>
+
+    <build>
+        <plugins>
```

Comment:
Use `maven-site-plugin` will create a typical maven document, it does not meet our brand style.

It is better if you can follow this [example project](https://github.com/asciidoctor/asciidoctor-maven-examples/tree/main/asciidoc-to-html-example) to create simple HTML5 site, then add styling later (in another ticket)

Check it put and run in local to see how it works, also review its document structure.

Current structure is not good. Should split document folder as `functional` instead of technical name (ansible, python), also make a boundary between user guide an development document